docs/architecture.md

```diff
@@ -54,6 +54,11 @@ For example:
 
 6. The MicroProfile Language Server receives the project information, adds it to its cache, and returns the completion options stored in the project information as the response to the original `textDocument/completion` request.
 
+In VS Code, the MicroProfile JDT is used as the external component, so the following diagram represents the communication sequence between components
+
+![Completion cache hit sequence](./images/property-completion-cache-hit-sequence.png)
```

Comment:
I love your diagram 👍 

But I think we should change the order of the images (to follow the real process because the first time cache is empty)

Reply:
Thanks! Re-ordered them to put the cache miss first.